datasets/us/dos_terrorist_exclusion/crawler.py

```diff
@@ -0,0 +1,54 @@
+from zavod import Context, helpers as h
+import requests
+from typing import cast
+from lxml import html
+
+HEADERS = {
+    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.4.1 Safari/605.1.15",
+}
```

Comment:
if this needs to be overwritten, do so using the metadata `http` config and see if you can make it work with `(zavod, opensanctions.org)` at the end so they can reach us if we're hurting them

Reply:
I was able to run with `fetch_html`, but if I remove this header it doesn't work. Do you mean adding `(zavod, opensanctions.org)` to the user agent? Like this?

```
Mozilla/5.0 (Macintosh; Intel ... (zavod, opensanctions.org)
```

Reply:
I actually meant like https://github.com/opensanctions/opensanctions/blob/main/datasets/_global/interpol/interpol_red_notices.yml#L42 but I see their website now has some kind of bot blocking that needs more than user agent to bypass, so I've switched to using zyte https://github.com/opensanctions/opensanctions/pull/932/commits/455eec80a1e68da6517b09765989a457892917c8